app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
There are no requirements, but it is a good habit to use typing
```suggestion
def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```